content/ai/ai.md

```diff
@@ -1,43 +1,35 @@
 
-In 1914, Spanish engineer [Leonardo Torres y Quevedo](https://en.wikipedia.org/wiki/Leonardo_Torres_y_Quevedo) demonstrates the first chess-playing machine in Paris, capable of receiving information about a chess game and playing a king and rook endgame against king from any position without the aid of human intervention.
+In 1914, Spanish engineer [Leonardo Torres y Quevedo](https://en.wikipedia.org/wiki/Leonardo_Torres_y_Quevedo) demonstrates the first chess-playing machine in Paris, capable of receiving information about a chess game and playing a king and rook endgame against the king from any position without the aid of human intervention.
 
-In 1950, [Alan Turing](https://en.wikipedia.org/wiki/Alan_Turing) publishes _Computing Machinery and Intelligence_, introducing with it the concept of the "imitation game". This would later be known as the Turing Test, which tests a machine's ability to display behavior and produce output that is indistinguishable from that of an adult person.
+In 1950, [Alan Turing](https://en.wikipedia.org/wiki/Alan_Turing) publishes _Computing Machinery and Intelligence_, introducing with it, the concept of the "imitation game". This would later be known as the Turing Test, which tests a machine's ability to display behavior and produce output that is indistinguishable from that of an adult person.
 
 The years ranging from 1956-1974 are considered the renaissance period of artificial intelligence with developments such as semantic nets, allowing machines to solve algebra word problems, and search algorithms that allowed machines to approach solving problems much like solving a maze.
 
-Following this period the field of AI experienced a sequence of years in lulls and bursts in progress between the years 1974 and 2011 where computing power and amounts of available data would be considerable bottlenecks. This period ends around 2011 where the advents of machine learning and deep learning have demonstrated great leaps in the field, producing agents such as [Deep Blue](<https://en.wikipedia.org/wiki/Deep_Blue_(chess_computer)>) and [AlphaGo](https://en.wikipedia.org/wiki/AlphaGo), computer programs that are able to beat some of the best human board game players in the world.
+Following this period, the field of AI experienced a sequence of years of lulls and bursts in progress between the years 1974 and 2011 where computing power and amounts of available data would be considerable bottlenecks. This period ends around 2011 when the advents of machine learning and deep learning demonstrated great leaps in the field, producing agents such as [Deep Blue](<https://en.wikipedia.org/wiki/Deep_Blue_(chess_computer)>) and [AlphaGo](https://en.wikipedia.org/wiki/AlphaGo), computer programs that can beat some of the best human board game players in the world.
```

Comment:
```suggestion
Following this period, the field of AI experienced lulls and bursts of progress (between the years 1974 and 2011) where computing power and the amount of available data would be considerable bottlenecks. This period ends around 2011 with the development of agents such as [Deep Blue](<https://en.wikipedia.org/wiki/Deep_Blue_(chess_computer)>) and [AlphaGo](https://en.wikipedia.org/wiki/AlphaGo), that were capable of matching and exceeding the best human board game players in the world.
```

Reply:
Some edits to simplify language. Also I think the (original) last sentence is technically incorrect. Deep learning and machine learning, more broadly, were not "introduced" in the 2000s. These algorithms actually go back many decades. The breakthroughs of that period were more a function of the improvement in computation in support of these existing methods. I've rephrased the last sentence to address this.